CoastalResilienceNetwork/GeositeFramework
# Setup text for translation

This is a first pass at setting up text in the UI so that it can be
translated.

* Wrap UI text in the core framework and plugins that should be
translated.
* Add text to en.json files for core framework and plugins.

**Testing instructions**
- Verify that a sample of the translations are set-up correctly.
- Look for areas in the app that may have been missed.

Connects to #598
